mechanical/assembly_renderer.py

```diff
@@ -71,20 +69,26 @@ def generate(self) -> cq.Assembly:
             cq_part = cq.importers.importStep(part.step_file)
             for tag in part.tags:
                 cq_part = cq_part.tag(tag)
-            assembly.add(cq_part, name=part.name, loc=cq.Location(part.position))
+            #Pylint appears to be confused by the multimethod __init__ used by cq.Location
```

Comment:
Multimethods are a large part of the way CadQuery works now, so this is not likely to get any better.

Reply:
Thanks. Yeah I am not worried about that. I was trying to just make a comment to make it clear why in the next line I was disabling a pylint warning.